lib/WUI/espif.cpp

```diff
@@ -98,10 +98,10 @@ enum MessageType {
 
 #if PRINTER_IS_PRUSA_XL
 // ESP32 FW version
-static const uint32_t SUPPORTED_FW_VERSION = 10;
+static const uint8_t SUPPORTED_FW_VERSION = 10;
```

Comment:
While we're at it, we could also make it `constexpr`

Reply:
Done as well